### DISCUSSION TITLE
Is there any paper to be referenced in?

### DISCUSSION THREAD

**Discussion by muazhari** [DataCanvasIO/HyperTS#75] [Q&A]
I want to use this library for academic purposes. But, I seem not found anything to be cited. Is there any legit paper provided that explains this library?

**Comment by zhangxjohn** [comment]
Thank you for your attention, there are currently no academic papers published on HyperTS. If you use HyperTS in your research, please cite us as follows:

Xiaojing Zhang, Jian Yang, Haifeng Wu, and Gang He. "HyperTS: A Full-Pipeline Automated Time Series Analysis Toolkit," https://github.com/DataCanvasIO/HyperTS, 2022. Version 0.2.0.
